src/or/policies.c

```diff
@@ -3038,11 +3040,12 @@ getinfo_helper_policies(control_connection_t *conn,
     SMARTLIST_FOREACH(configured_addresses, tor_addr_t *, a, tor_free(a));
     smartlist_free(configured_addresses);
   } else if (!strcmpstart(question, "exit-policy/")) {
-    const routerinfo_t *me = router_get_my_routerinfo();
-
     int include_ipv4 = 0;
     int include_ipv6 = 0;
 
+    const routerinfo_t *me = NULL;
+    int err = router_get_my_routerinfo_with_err((routerinfo_t **)&me);
```

Comment:
Hmmm I would go the other way around here so we don't have this cast violating the `const`.

Make `router_get_my_routerinfo_with_err()` return the `me` const pointer and pass a `int` pointer for the err. If your `me` is NULL, then we know to look at the `err`.

Reply:
Fixed in 36f7d0a940bbe8695d5343fccc1a0560a529883a.